components/script/dom/window.rs

```diff
@@ -1445,6 +1451,12 @@ impl Window {
         None
     }
 
+    pub fn freeze(&self) {
+        self.upcast::<GlobalScope>().suspend();
+        // A hint to the JS runtime that now would be a good time to GC this window.
+        self.Gc();
```

Comment:
This window will not be GCed because it is being accessed by a root that will prevent it being GCed. I'm not sure that this hint makes sense here.